src/timeline/components/stateChronicles/TimelineChronicleLegend.vue

```diff
@@ -108,6 +123,8 @@ export default {
             displayedValue: "",
             endPoints: null,
             limits: null,
+            numericHeight: 40,
```

Comment:
Could we use a constant at the top of the script section for the numericHeight? Its easier to spot when we are trying to change it.